test/driver-test/driver-test.cpp

```diff
@@ -0,0 +1,312 @@
+/*
+* driver-test.cpp
+* Nick DeMarco
+* (c) 2020 sigcpp https://sigcpp.github.io. See LICENSE.MD
+*
+* Attribution and copyright notice must be retained.
+* - Attribution may be augmented to include additional authors
+* - Copyright notice cannot be altered
+* Attribution and copyright info may be relocated but they must be conspicuous.
+*
+* Tester for test-stl-lite driver
+*/
+
+#include "../verifiers.h"
+#include "../options.h"
+#include "../tester.h"
+#include "../utils.h"
+
+#include <iostream>
+#include <string_view>
+#include <vector>
+
+void driver_test();
+void test_get_options();
+void reset_default_options(Options& o);
+void compare(const Options& received, const Options& expected, const std::string& test_case);
+std::vector<std::pair<std::string, Options>> generate_cmdline_option_pairs();
+
+void driver_test()
+{
+	test_get_options();
+}
+
+void test_get_options()
+{
+	char* args[100];
```

Comment:
This declaration begs a comment. We know what it does. We don't know why this data exists, and why the bound of 100 is chosen.